### PR TITLE
MLI file names conform to snake case

### PR DESCRIPTION
**Pull request by AlyssaCote**
In this PR I updated the non snake case filenames to be snake case.

### CODE REVIEW

File: ex/high_throughput_inference/standalone_workermanager.py [hunk 1]
```diff
@@ -58,17 +58,17 @@
 
 from smartsim._core.entrypoints.service import Service
 from smartsim._core.mli.comm.channel.channel import CommChannelBase
-from smartsim._core.mli.comm.channel.dragonchannel import DragonCommChannel
-from smartsim._core.mli.comm.channel.dragonfli import DragonFLIChannel
-from smartsim._core.mli.infrastructure.storage.dragonfeaturestore import (
+from smartsim._core.mli.comm.channel.dragon_channel import DragonCommChannel
```
Comment (mellis13) [inline ex/high_throughput_inference/standalone_workermanager.py:61]: Should this file be standalone_worker_manager?

**Review by mellis13 — COMMENTED** [review]
Just one comment/question

File: ex/high_throughput_inference/standalone_workermanager.py (hunk 1 above)
Comment (AlyssaCote) [reply ex/high_throughput_inference/standalone_workermanager.py:61]: Oops, yes! Fixed.

**Review by mellis13 — APPROVED** [review]
thank_you!

**Pull request merged**